### ISSUE TITLE
Bazel on Fedora 33

### ISSUE THREAD

**Issue by Lafred** [bazelbuild/bazel#13470]
Copr repo for bazel owned by vbatts

Trying to build the cpp tutorial for bazel 4.

**Comment by Lafred** [comment]
# Current Work Around
bazel build  --spawn_strategy=standalone //main:hello-world

**Comment by aiuto** [comment]
Can you clarify this by adding an explicit declaration of what you are attempting, what is the problem and how to reproduce it.

**Comment by Lafred** [comment]
What do you mean by explicit. Can you give me an example?

**Comment by Lafred** [comment]
I downloaded Bazel on Fedora33 by 

> sudo dnf copr enable vbatts/bazel
> sudo dnf install bazel4

I then attempted to build the example downloaded from the Bazel website.  Following instructions here:

https://docs.bazel.build/versions/4.0.0/tutorial/cpp.html

On a bazel build inside "/home/<user>/Repos/bazel-examples/cpp-tutorial/stage1", I run bazel build.

I get the following error.

bazel build //main:hello-world
Starting local Bazel server and connecting to it...
INFO: Analyzed target //main:hello-world (15 packages loaded, 52 targets configured).
INFO: Found 1 target...
ERROR: /home/jameselis/Repos/bazel-examples/cpp-tutorial/stage1/main/BUILD:3:10: Compiling main/hello-world.cc failed: (Exit 1): gcc failed: error executing command /usr/lib64/ccache/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF ... (remaining 20 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox gcc failed: error executing command /usr/lib64/ccache/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++0x' -MD -MF ... (remaining 20 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
ccache: error: Failed to create temporary file for /run/user/1000/ccache-tmp/tmp.cpp_stdout.DDwjbn: Read-only file system
Target //main:hello-world failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 4.513s, Critical Path: 0.05s
INFO: 5 processes: 5 internal.
FAILED: Build did NOT complete successfully
